src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -2156,6 +2156,21 @@ def test_feature_unregister(self):
             self.check_pattern('properties.state', 'Unregistering|Unregistered')
         ])
 
+    @AllowLargeResponse(8192)
+    def test_feature_registration_list(self):
+        self.cmd('feature registration list', checks=self.check("length([?name=='Microsoft.Network/SkipPseudoVipGeneration'])", 1))
+
+        self.cmd('feature registration show --provider-namespace Microsoft.Network -n AllowLBPreview')
+    
+    @AllowLargeResponse(8192)
+    def test_feature_registration_create(self):
+        self.cmd('feature registration create --namespace Microsoft.Network --name AllowLBPreview', checks=[
+            self.check_pattern('properties.state', 'Registering|Registered')
+        ])
+
+    @AllowLargeResponse(8192)
+    def test_feature_registration_delete(self):
+        self.cmd('feature registration delete --namespace Microsoft.Network --name AllowLBPreview')
```

Comment:
I think the CI issue caused by adding `confirmation=True` is caused by not adding `--yes` to the delete command in this test.
```suggestion
        self.cmd('feature registration delete --namespace Microsoft.Network --name AllowLBPreview' --yes)
```

Reply:
yes, it worked with '--yes'